server/app.py

```diff
@@ -12,4 +12,77 @@
 app.register_blueprint(ping_handler)
 
 db = SQLAlchemy(app)
-migrate = Migrate(app, db)
\ No newline at end of file
+migrate = Migrate(app, db)
+
+from models import List, Product
+
+@app.route('/create_list', methods=['POST'])
+def createList():
+    if request.method == 'POST':
+        body = json.loads(request.get_data())
+        list_item = List(body['product_type'])
+        db.create_all()
+        db.session.add(list_item)
+        try:
+            db.session.commit()
+        except:
+            return jsonify({'response': "{} already exists in the database".format(body['product_type'])}), 400
+    return jsonify({'response': "{} was successfully added to the database".format(body['product_type'])}), 200
+
+
+@app.route('/add_product', methods=['GET', 'POST'])
+def addProduct():
+    if request.method == "GET":
+        product_types = List.query.all()
+        ls = []
+        for item in product_types:
+            ls.append(
+                {
+                    "id": item.id,
+                    "product_type": item.product_type
+                })
+        return jsonify({'list': ls}), 200
+
+    elif request.method == 'POST':
+        body = json.loads(request.get_data())
+        product_item = Product(
+            body['list_id'], body['name'], body['description'], body['price'], body['url'])
```

Comment:
Let's add some error checking - what if the incoming request doesn't have all of these fields?
Also, maybe not in this PR, but we may want to think about adding data validation, like price needs to be a float, etc

Reply:
so by default all the input is of type string, I only did validation for ` body['price']` because this requires a 2-decimal float. `body['list_id']` I casted it to type int().